jekyll/_cci2/testing-llm-enabled-applications-through-evaluations.adoc

```diff
@@ -65,7 +65,7 @@ Given the volatile nature of evaluations, evaluations orchestrated by the Circle
 
 Instead, a summary of the evaluation results is created and presented:
 
-* As a comment on the corresponding GitHub pull request (GitHub projects only):
+* As a comment on the corresponding GitHub pull request (currently available only for projects integrated with Github OAuth):
```

Comment:
```suggestion
* As a comment on the corresponding GitHub pull request (currently available only for projects integrated with xref:github-integration#[Github OAuth]):
```

Reply:
just a suggestion so people can get some more info on what this actually means

Reply:
accepted!